packages/components/psammead-most-read/src/list.stories.jsx

```diff
@@ -0,0 +1,104 @@
+    href: 'https://www.bbc.com/vietnamese/institutional-49283563',
+  },
+  {
+    title: 'John Lewis staff bonus cut again as profits fall',
+    href: 'https://www.bbc.com/vietnamese/institutional-49283563',
+  },
+  {
+    title: 'John Lewis staff bonus cut again as profits fall',
+    href: 'https://www.bbc.com/vietnamese/institutional-49283563',
+  },
+  {
+    title: 'John Lewis staff bonus cut again as profits fall',
+    href: 'https://www.bbc.com/vietnamese/institutional-49283563',
+  },
+  {
+    title: 'John Lewis staff bonus cut again as profits fall',
+    href: 'https://www.bbc.com/vietnamese/institutional-49283563',
+  },
+  {
+    title: 'John Lewis staff bonus cut again as profits fall',
+    href: 'https://www.bbc.com/vietnamese/institutional-49283563',
+  },
+  {
+    title: 'John Lewis staff bonus cut again as profits fall',
+    href: 'https://www.bbc.com/vietnamese/institutional-49283563',
+  },
+  {
+    title: 'John Lewis staff bonus cut again as profits fall',
+    href: 'https://www.bbc.com/vietnamese/institutional-49283563',
+  },
+  {
+    title: 'John Lewis staff bonus cut again as profits fall',
+    href: 'https://www.bbc.com/vietnamese/institutional-49283563',
+  },
+];
+
+const itemsRTL = [
+  {
+    title: "ایران از لغو 'رزمایش قطع اینترنت' خبر داد",
+    href: 'https://www.bbc.com/vietnamese/institutional-49283563',
+  },
+  {
+    title: "ایران از لغو 'رزمایش قطع اینترنت' خبر داد",
+    href: 'https://www.bbc.com/vietnamese/institutional-49283563',
+  },
+  {
+    title: "ایران از لغو 'رزمایش قطع اینترنت' خبر داد",
+    href: 'https://www.bbc.com/vietnamese/institutional-49283563',
+  },
+  {
+    title: "ایران از لغو 'رزمایش قطع اینترنت' خبر داد",
+    href: 'https://www.bbc.com/vietnamese/institutional-49283563',
+  },
+  {
+    title: "ایران از لغو 'رزمایش قطع اینترنت' خبر داد",
+    href: 'https://www.bbc.com/vietnamese/institutional-49283563',
+  },
+  {
+    title: "ایران از لغو 'رزمایش قطع اینترنت' خبر داد",
+    href: 'https://www.bbc.com/vietnamese/institutional-49283563',
+  },
+  {
+    title: "ایران از لغو 'رزمایش قطع اینترنت' خبر داد",
+    href: 'https://www.bbc.com/vietnamese/institutional-49283563',
+  },
+  {
+    title: "ایران از لغو 'رزمایش قطع اینترنت' خبر داد",
+    href: 'https://www.bbc.com/vietnamese/institutional-49283563',
+  },
+  {
+    title: "ایران از لغو 'رزمایش قطع اینترنت' خبر داد",
+    href: 'https://www.bbc.com/vietnamese/institutional-49283563',
+  },
+  {
+    title: "ایران از لغو 'رزمایش قطع اینترنت' خبر داد",
+    href: 'https://www.bbc.com/vietnamese/institutional-49283563',
+  },
+];
+
+const stories = storiesOf(STORY_KIND, module)
+  .addDecorator(withKnobs)
+  .addDecorator(withServicesKnob());
+
+stories.add(`MostReadList LTR`, ({ script, service }) => (
+  <MostReadList items={itemsLTR} service={service} script={script} />
+));
+
+stories.add(`MostReadList RTL`, ({ script, service }) => (
+  <MostReadList items={itemsRTL} service={service} script={script} />
```

Comment:
You'll want to pass in the direction prop of value `'rtl'` to the component
```suggestion
  <MostReadList dir="rtl" items={itemsRTL} service={service} script={script} />
```

Reply:
Resolved